oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/web/DefaultOAuth2AuthorizationRequestResolverTests.java

```diff
@@ -105,7 +105,7 @@ public void resolveWhenAuthorizationRequestWithValidClientThenResolves() {
 				.isEqualTo("http://localhost/login/oauth2/code/" + clientRegistration.getRegistrationId());
 		assertThat(authorizationRequest.getScopes()).isEqualTo(clientRegistration.getScopes());
 		assertThat(authorizationRequest.getState()).isNotNull();
-		assertThat(authorizationRequest.getAdditionalParameters())
```

Comment:
Since this is to fix gh-5940 we should assert that the additional parameters do not contain the registration id (just because it is in the attributes -- the next assertion -- doesn't mean it isn't also in the parameters)

Reply:
Makes sense. Applied and pushed.